flake.nix

```diff
@@ -46,7 +46,7 @@
 
       checks = forAllSystems (system: {
         tarball = jobs.${system}.tarball;
-      } // lib.optionalAttrs (self.legacyPackages.${system}.stdenv.isLinux) {
+      } // lib.optionalAttrs (self.legacyPackages.${system}.stdenv.isLinux && !self.legacyPackages.${system}.targetPlatform.isPower64) {
```

Comment:
Please add a comment explaining what's broken here so it can hopefully be removed in the future.

Reply:
Done